bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -142,7 +142,13 @@ func (coord *coordinator) getPackageManagers(kfdef *kfdefsv3.KfDef) *map[string]
 // kftypesv3.LoadKfApp which will try and dynamically load a .so
 //
 func getPackageManager(kfdef *kfdefsv3.KfDef) (kftypesv3.KfApp, error) {
-	switch kfdef.Spec.PackageManager {
+
+	packageManager := kfdef.Spec.PackageManager
```

Comment:
Why are there kfctl changes in this PR?

Reply:
@jlewi these are minor fixes which became necessary after the new semantics PR was merged.
I have opened a separate PR to add them, but I also include them here to get the tests working.
https://github.com/kubeflow/kubeflow/pull/4244

Reply:
Got it. Thanks